package.json

```diff
@@ -90,6 +90,9 @@
           "polyfill": false,
           "regenerator": true
         }
+      ],
+      [
+        "transform-class-properties"
```

Comment:
this is for the class properties to work with babel; especially useful for bound functions, but some things could be moved from constructors to there, like initial state.